feature/news/src/main/java/com/teamwable/news/match/NewsMatchScoreAdapter.kt

```diff
@@ -0,0 +1,32 @@
+package com.teamwable.news.match
+
+import android.os.Build
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.annotation.RequiresApi
+import androidx.recyclerview.widget.ListAdapter
+import com.teamwable.model.NewsMatchScoreModel
+import com.teamwable.news.databinding.ItemNewsMatchScoreBinding
+import com.teamwable.ui.extensions.ItemDiffCallback
+
+class NewsMatchScoreAdapter : ListAdapter<NewsMatchScoreModel, NewsMatchScoreViewHolder>(
+    NewsMatchScoreAdapterDiffCallback,
+) {
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): NewsMatchScoreViewHolder {
+        val binding = ItemNewsMatchScoreBinding.inflate(LayoutInflater.from(parent.context), parent, false)
+        return NewsMatchScoreViewHolder(binding)
+    }
+
+    @RequiresApi(Build.VERSION_CODES.O)
+    override fun onBindViewHolder(holder: NewsMatchScoreViewHolder, position: Int) {
+        holder.bind(currentList[position])
```

Comment:
```suggestion
        holder.bind(getItem(position))
```
큰 차이는 없는데 데이터의 변경이 많다면 getItem이 조금 더 효율적이라고 합니다!

Reply:
오호라 접수!